extendeddatasquare.go

```diff
@@ -232,3 +232,49 @@ func deepCopy(original [][]byte) [][]byte {
 func (eds *ExtendedDataSquare) Width() uint {
 	return eds.width
 }
+
+// FlattenedEDS returns the extended data square as a flattened slice of bytes.
+func (eds *ExtendedDataSquare) FlattenedEDS() [][]byte {
+	return eds.Flattened()
+}
```

Comment:
Should we just remove it and keep Flattened?

Reply:
Since Flattened is implemented on data square, I think the godoc will be a little confusing...

```go
// Flattened returns the concatenated rows of the data square.
func (ds *dataSquare) Flattened() [][]byte {
```

it doesn't specify ODS or EDS and in fact it can't b/c it doesn't know

Reply:
Good idea @walldiss ! Will do.

Reply:
Resolved via 74da9a2cf48d358ff697060002a45fb843fc334d